data/orders.yml

```diff
@@ -117,12 +117,12 @@
   state: planning
 - requesting_agency: Department of Agriculture
   office: Forest Service
-  rfq_id: null
+  rfq_id: 1160564
   title: Forest Service e-Permitting
-  solicitation_date: (2016-11)
-  period_of_performance: TBD
-  repository: null
-  description: null
+  solicitation_date: 2016-12-21
+  period_of_performance: Likely Jan-Feb 2017
```

Comment:
From the other postings, this field is used for the length of the period (e.g. "base period of 1 month, with 2 option periods of 1 month each") rather than the dates of the period.